pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -31,8 +32,8 @@ in stdenvNoCC.mkDerivation {
       })
     else
       (fetchurl {
-        url = "https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15816/l_mkl_${version}.tgz";
-        sha256 = "0zkk4rrq7g44acxaxhpd2053r66w169mww6917an0lxhd52fm5cr";
+        url = "http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/16318/l_mkl_${version}.tgz";
```

Comment:
Thank you for doing this!  I've been trying from time to time for weeks now, but the Intel download site kept refusing to give me the download link [1].  

Anyways, any reason why we have to move to HTTP instead of HTTPS?

[1] https://software.intel.com/en-us/forums/intel-math-kernel-library/topic/841834